cryptography/src/commonTest/kotlin/com/wire/kalium/cryptography/E2EIClientTest.kt

```diff
@@ -35,7 +35,7 @@ class E2EIClientTest : BaseMLSClientTest() {
     }
 
     private suspend fun createE2EIClient(user: SampleUser): E2EIClient {
-        return createMLSClient(user.qualifiedClientId).e2eiNewActivationEnrollment(
+        return createMLSClient(user.qualifiedClientId, listOf(1.toUShort()), 1.toUShort()).e2eiNewActivationEnrollment(
```

Comment:
suggestion: define these values then we can change them later easier?

Reply:
i was thinking about a way to test all possible ciphersuits